### PR TITLE
translate sort link

### PR DESCRIPTION
**Pull request by Johnschmidfront**

### CODE REVIEW

**Comment by Johnschmidfront** [comment]
@xi @local-girl please review.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/i18n.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ translateMe = [
     "TR__PUBLISH",
     "TR__RATES",
     "TR__SHARE",
+    "TR__SORT",
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/i18n.ts]: This is not necessary

**Comment by xi** [comment]
The english translation is missing.

**Comment by xi** [comment]
LGTM.

**Pull request merged**